.github/workflows/preview.yml

```diff
@@ -82,3 +82,21 @@ jobs:
           header: lighthouse
           message: |
             ${{ steps.lighthouse_result.outputs.comment }}
+  e2e:
```

Comment:
add e2e to the preview workflow so that we run e2e's after the deployment is finished.